bgcval2/UKESMpython.py

```diff
@@ -463,6 +463,59 @@ def shouldIMakeFile(fin, fout, debug=True):
     return False
 
 
+
+
+def load_coords_from_netcdf(mdfile):
+    """
+    Looks in mdfile to see what the coords look like.
+    """
+    # Set input coordinates:
+    coord_candidates = {
+        't': ['time_centered','time', 'index_t', 'time_counter'],
```

Comment:
This is kinda weird. Surely there are other values here? maybe these should be loaded from yml files?